internal/math32/sqrt_arm64.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Copyright ©2015 The Gonum Authors. All rights reserved.
```

Comment:
2020?

Reply:
It's a copy of the sqrt_amd64.go file without changes, so I think not.